tests/libkubevirt/config/kvconfig.go

```diff
@@ -51,26 +51,43 @@ import (
 
 type compare func(string, string) bool
 
-func RegisterKubevirtConfigChange(change func(c v1.KubeVirtConfiguration) (*patch.PatchSet, error)) error {
+type KvChangeOption func(kv v1.KubeVirt) *patch.PatchSet
```

Comment:
One might ask why the whole KubeVirt struct and not just the spec.
I personally think its a good idea, as it enables extension of this functionality to also modify metadata fields, such as annotations.
Please consider giving a few words about it in the commit message.